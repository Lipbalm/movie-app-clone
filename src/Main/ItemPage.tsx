import React, { FC } from "react";
import { RouteComponentProps } from "react-router-dom";
import { IMatchParams } from "../Modules/Interfaces";
import { BodyWrapper } from "./MainStyle";

const ItemPage: FC<RouteComponentProps<IMatchParams>> = ({ match }) => {
  const { id } = match.params;
  return <BodyWrapper>ItemPage : {id}</BodyWrapper>;
};

export default ItemPage;
