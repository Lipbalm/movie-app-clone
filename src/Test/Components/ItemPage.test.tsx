import React from "react";
import { render } from "@testing-library/react";
import { createMemoryHistory } from "history";
import { Router } from "react-router-dom";
import ItemPage from "../../Main/ItemPage";

const setup = () => {
  const history = createMemoryHistory();
  return {
    ...render(
      <Router history={history}>
        <ItemPage />
      </Router>
    )
  };
};

describe("<ItemPage />", () => {
  it("snapshot", () => {
    const { container } = setup();
    expect(container).toMatchSnapshot();
  });
});
