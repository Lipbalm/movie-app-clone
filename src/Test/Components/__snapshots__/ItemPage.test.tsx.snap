// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemPage /> snapshot 1`] = `
<div>
  <div>
    ItemPage
  </div>
</div>
`;
